styx-common/src/test/java/com/spotify/styx/storage/DatastoreStorageTest.java

```diff
@@ -636,6 +647,37 @@ public void shouldStoreAndReadBackfill(String reverse) throws Exception {
     assertThat(storage.getBackfill(backfill.id()), equalTo(Optional.of(backfill)));
   }
 
+  @Test
+  public void shouldReadBackfillWithMissingReverseField() throws Exception {
+    final Backfill backfill = Backfill.newBuilder()
+        .id("backfill-2")
+        .start(Instant.parse("2017-01-01T00:00:00Z"))
+        .end(Instant.parse("2017-01-02T00:00:00Z"))
+        .workflowId(WorkflowId.create("component", "workflow2"))
+        .concurrency(2)
+        .nextTrigger(Instant.parse("2017-01-01T00:00:00Z"))
+        .schedule(DAYS).build();
+
+    final Key key = DatastoreStorage.backfillKey(datastore.newKeyFactory(), backfill.id());
+    Entity.Builder builder = Entity.newBuilder(key)
+        .set(PROPERTY_CONCURRENCY, backfill.concurrency())
+        .set(PROPERTY_START, instantToTimestamp(backfill.start()))
+        .set(PROPERTY_END, instantToTimestamp(backfill.end()))
+        .set(PROPERTY_COMPONENT, backfill.workflowId().componentId())
+        .set(PROPERTY_WORKFLOW, backfill.workflowId().id())
+        .set(PROPERTY_SCHEDULE, backfill.schedule().toString())
+        .set(PROPERTY_NEXT_TRIGGER, instantToTimestamp(backfill.nextTrigger()))
+        .set(PROPERTY_ALL_TRIGGERED, backfill.allTriggered())
+        .set(PROPERTY_HALTED, backfill.halted());
+
+    backfill.description().ifPresent(x -> builder.set(PROPERTY_DESCRIPTION, StringValue
```

Comment:
Is this to make test case more complete? `description` is empty by definition.

Reply:
No reason really, it was just a quick whip up meant to be a hotfix, but resorted to holding it off a bit. I can remove that for better cleanliness.